models/twilio_number_overview.sql

```diff
@@ -1,15 +1,15 @@
-with message_enhanced as (
+with message_enhanced as ( 
```

Comment:
Reminder that if we add the disable config for the message models then we will need to add a disable config block here.

Again, I am starting to lean to removing that config as now both these models will be removed if the customer doesn't have messaging. I don't see the value in this package if messaging is not used.

Reply:
Pending earlier conversation thread

Reply:
Yeah let's plan to remove the config for just messages, but we can add one for messaging_service and account for that accordingly.